Datahub.Core/Components/DHMarkdown.razor

```diff
@@ -3,27 +3,55 @@
 
 @if (Content != null)
 {
-    <span @ref="_elementReference" class="markdown-container">
+    <div @ref="_elementReference" @attributes=@InputAttributes>
         <MudMarkdown 
-            Value=@Content 
+            Value=@FinalContent 
             OverrideLinkUrl=@OverrideLinkUrl 
             OverrideHeaderTypo=@SubstituteHeaderTypo />
-    </span>
+    </div>
 }
 
 @code {
+    private const string MARKDOWN_CONTAINER_CLASS = "markdown-container";
 
     [Parameter]
     public string Content { get; set; }
 
     [Parameter]
     public Func<string, string> LinkRewriter { get; set; }
 
+    [Parameter]
+    public List<(string, string)> Substitutions { get; set; }
+
     [Parameter(CaptureUnmatchedValues = true)]
     public Dictionary<string, object> InputAttributes { get; set; }
 
     private ElementReference _elementReference;
 
+    private string _finalContent;
+    private bool _needsRefresh = true;
+
+    private string FinalContent
+    {
+        get
+        {
+            if (_needsRefresh)
+            {
+                if (Substitutions?.Count > 0)
+                {
+                    _finalContent = Substitutions.Aggregate(Content, (current, s) => current.Replace(s.Item1, s.Item2));
+                }
+                else
+                {
+                    _finalContent = Content;
+                }
+                _needsRefresh = false;
+            }
+
+            return _finalContent;
+        }
+    }
+
```

Comment:
The cached _finalContent does get refreshed whenever substitutions are updated, however for some reason the MudMarkdown element does not re-render it with the updated content. It did previously during the dev cycle (possibly only when the pages now using <EmbeddedResources> had the original <Resources> component). It needs some investigation.